### PR TITLE
feat: add `workflow_run:completed` webhook, set empty `client_payload` for `workflow_dispatch` webhook

### PR DESCRIPTION
**Pull request by heuels**
Hi!

Just a minor thing, but it still looks worth fixing while I battle the JSON Schema :)

### CODE REVIEW

File: index.json [hunk 1]
```diff
@@ -20593,16 +20593,13 @@
   },
   {
     "name": "repository_dispatch",
-    "description": "This event occurs when a GitHub App sends a `POST` request to the \"[Create a repository dispatch event](https://docs.github.com/en/v3/repos/#create-a-repository-dispatch-event)\" endpoint.\n\nGitHub Apps must have the `contents` permission to receive this webhook.",
+    "description": "This event occurs when a GitHub App sends a `POST` request to the \"[Create a repository dispatch event](https://docs.github.com/en/v3/repos/#create-a-repository-dispatch-event)\" endpoint.",
```
Comment (gr2m) [inline index.json:20596]: the description is directly parsed from the website, this change will be reverted with next next automated update

Comment (heuels) [reply index.json:20596]: I haven't edited `index.json` manually – I ran `bin/octokit-webhooks.js update` and that's what was generated.

Comment (heuels) [reply index.json:20596]: As far as I can see, they've recently moved this note to a separate section: https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#availability-35.

**Comment by heuels** [comment]
@gr2m, the schedule section of the `update` workflow is currently commented out:
https://github.com/octokit/webhooks/blob/b4ece1185bcb3a6737e93b536867052fa75f4cee/.github/workflows/update.yml#L2-L4

**Comment by gr2m** [comment]
oopsies, that should have been uncommented as part of https://github.com/octokit/webhooks/pull/134.

When you ran `bin/octokit-webhooks.js update`, it should have updated the cache, too. Could you commit that file? And also uncomment the lines in webhooks/.github/workflows/update.yml?

**Comment by heuels** [comment]
Done 🙂

**Pull request merged**